docs/ErrorHandlingRationale.md

```diff
@@ -1,5 +1,62 @@
 # Error Handling Rationale and Proposal
 
+- [Error Handling Rationale and Proposal](#error-handling-rationale-and-proposal)
```

Comment:
I don’t think this one is necessary.

Reply:
While the tests are failing I will take this opportunity to suggest adding a heading to the table, as in [GettingStarted.md](https://github.com/apple/swift/blob/main/docs/HowToGuides/GettingStarted.md) or [ARCOptimization.md](https://github.com/apple/swift/blob/main/docs/ARCOptimization.md)